server/src/main/java/org/opensearch/search/builder/SearchSourceBuilder.java

```diff
@@ -418,7 +418,7 @@ public QueryBuilder postFilter() {
      */
     public SearchSourceBuilder from(int from) {
         if (from < 0) {
-            throw new IllegalArgumentException("[from] parameter cannot be negative");
+            throw new IllegalArgumentException("[from] parameter cannot be negative, found [" + from + "]");
```

Comment:
Nit, I see more places (but not all) using a formatter, not sure if it's safer/better/cleaner. 

```
OpenSearchParseException("Expected a string but found [{}] instead", p.currentToken());
```